plugins/com.google.cloud.tools.eclipse.util.test/src/com/google/cloud/tools/eclipse/util/NatureUtilsTest.java

```diff
@@ -63,10 +63,9 @@ public void testRemoveNature() throws CoreException {
 
   @Test
   public void testRemoveNature_nonExistingNature() throws CoreException {
-    // By default, project has faceted project nature.
-    assertArrayEquals(new String[] {FACETED_NATURE_ID}, project.getDescription().getNatureIds());
+    assertEquals(0, project.getDescription().getNatureIds().length);
 
     NatureUtils.removeNature(project, JavaCore.NATURE_ID, monitor);
-    assertArrayEquals(new String[] {FACETED_NATURE_ID}, project.getDescription().getNatureIds());
+    assertEquals(0, project.getDescription().getNatureIds().length);
```

Comment:
This situation shouldn't happen, but it is kinda handy to have the natures shown when it fails?